gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -993,6 +1067,15 @@ MM_ParallelGlobalGC::internalPostCollect(MM_EnvironmentBase *env, MM_MemorySubSp
 #if defined(OMR_GC_IDLE_HEAP_MANAGER)
 	_extensions->lastGCFreeBytes = _extensions->heap->getApproximateActiveFreeMemorySize(MEMORY_TYPE_OLD);
 #endif
+
+
+#if defined(OMR_ENV_DATA64) && !defined(OMR_GC_COMPRESSED_POINTERS)
+	if (1 == (1 & _extensions->fvtest_enableReadBarrierVerification)) {
+		_delegate.poisonSlots(env);
```

Comment:
put a two short comments for each of these lines: 'poison root slots' and 'posion heap slots'. same for heal